pkg/providers/tinkerbell/create.go

```diff
@@ -133,6 +133,10 @@ func (p *Provider) SetupAndValidateCreateCluster(ctx context.Context, clusterSpe
 	writer := hardware.NewMachineCatalogueWriter(p.catalogue)
 	machineValidator := hardware.NewDefaultMachineValidator()
 
+	// TODO(chrisdoherty4) Build the selectors slice using the selectors from TinkerbellMachineConfig's
+	var selectors []hardware.MachineSelector
+	machineValidator.Register(hardware.MatchingDisksForSelectors(selectors))
```

Comment:
check err return here

Reply:
Sorry, I'm not tracking. What error needs checking?

Reply:
oh, i had thought machine validator's Register does have an error return, but i see that it doesn't